stable/etcd-operator/templates/cluster.yaml

```diff
@@ -1,6 +1,6 @@
-{{- if .Values.cluster.enabled -}}
-apiVersion: "coreos.com/v1"
-kind: "EtcdCluster"
+{{- if and .Values.cluster.enabled (not .Release.IsInstall) -}}
```

Comment:
And, the TPR is not required in install mode because the operator deployment is all that's necessary to install the etcd-operator. The operator itself installs the TPR, which after this occurs, it allows you to create the `Cluster` resource kind.